src/test/java/com/unzer/payment/business/paymenttypes/ApplepayTest.java

```diff
@@ -40,6 +42,17 @@
 
 public class ApplepayTest extends AbstractPaymentTest {
 
+    @Test
+    public void testCreateApplePayTypeFromJson() throws JsonProcessingException {
```

Comment:
Is this the certificate?

Reply:
This is the Response JSON from Apple after the Payment Authorization. But the Values have been masked on different parts.